react/features/base/storage/PersistenceRegistry.js

```diff
@@ -134,10 +149,13 @@ class PersistenceRegistry {
 
         for (const name of Object.keys(this._elements)) {
             if (state[name]) {
-                filteredState[name]
-                    = this._getFilteredSubtree(
-                        state[name],
-                        this._elements[name]);
+                const filteredSubtree = this._getFilteredSubtree(
+                    state[name],
+                    this._elements[name]);
+
+                if (typeof filteredSubtree !== 'undefined') {
```

Comment:
when can this be the case? AFAIS there is no way for _getFilteredSubtree to return undefined, is there?